processing/data_collection/gazette/spiders/base.py

```diff
@@ -18,6 +20,88 @@ def __init__(self, start_date=None, *args, **kwargs):
                 self.start_date = parsed_data.date()
 
 
+class DiarioMunicipalSigpubGazetteSpider(BaseGazetteSpider):
```

Comment:
Perhaps just `SigPubGazetteSpider` as the class name would be shorter and still meaningful.

Reply:
I wrote it that way because the name Sigpub is kind of hidden, so DiarioMunicipal could carry more meaning as it is the name of the website. But the name of the website is kind of generic then concatenating both seemed a good approach. But it is kind of big, yeah :/

I'll change it.

Reply:
The company behind it is Vox, this could also be in the name. DiarioMunicipal can be used by many states or even a city, too generic I guess.

Reply:
Feels a bit weird putting the company name, doesn't it? hehehhee